src/main/java/elevator/Elevator.java

```diff
@@ -0,0 +1,7 @@
+package elevator;
+
+public interface Elevator extends Comparable<Elevator>{
```

Comment:
@dwaynewade0033 please format the code.
Why we need `Comparable` here? I was suggesting to look into `Comparable` just to understand how to code direction possibilities.